Automattic/sensei
# Add basic integrations into Site Health

This is my first PR to integrate the Status and Tools plugin.

### Changes proposed in this Pull Request

* Adds very basic _Site Health_ integration.
* On _Status_ tab, adds a test to make sure learners are calculated (just goes into recommended if not).
* On _Info_ tab, adds basic debug information. **I added some template override version checking for themes.** We can use the methods for this later on for more advanced update nudging. Note: HTML formatting isn't allowed on values in this screen.

Example with Aardvark theme:
<img width="811" alt="Screen Shot 2020-12-09 at 10 28 59 pm" src="https://user-images.githubusercontent.com/68693/101696374-f15a3800-3a6d-11eb-9adf-7806c3eda48e.png">

Debug sample:
```
### sensei-lms ###

template_overrides: 
	archive-lesson.php: Mismatch (plugin v2.0.0; theme v1.9.0)
	content-course.php: Match (v1.12.2)
	content-lesson.php: Match (v1.9.0)
	course-results.php: Mismatch (plugin v3.6.0; theme v1.12.2)
	loop-course.php: Match (v1.9.0)
	loop-lesson.php: Match (v1.9.0)
	single-course.php: Mismatch (plugin v3.6.0; theme v1.12.2)
	single-course/lessons.php: Match (v1.9.0)
	single-lesson.php: Mismatch (plugin v3.6.0; theme v1.12.2)
	single-quiz.php: Mismatch (plugin v3.6.0; theme v2.0.0)
is_calculation_pending: Complete
legacy_enrolment: 1599474009
```

### Testing instructions

* Main thing to test is template version override checks. Switch to a theme that overrides Sensei LMS template files. Make sure they show up in Tools > Status > Info > Sensei LMS and have the correct template version check.
